### ISSUE TITLE
update server version

### ISSUE THREAD

**Issue by kimuss** [jsknnr/enshrouded-server#93]
Hello,
thank you for your work, I could start server on a LXC Proxmox.
I was wondering if this image will always be updated to the latest steam version.
thank you

**Comment by walisson7** [comment]
Just installed and used v2.1.6 on GoogleCloud, it works perfectly.

About your question, I think it will remain updated because the entrypoint runs steamcmd and it forces the install/update every startup, we will need an update just if game servers vars changes or in case of fixes into docker image.

Correct me if I'm wrong, kinda new to this... learning a lot just for fun :)

**Comment by jsknnr** [comment]
You are correct. steamcmd and the server files are updated when the container starts.
